bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -574,6 +576,8 @@ private void handlePlayersList(String message) {
                     }
                 }
 
+                seenAddresses.add(macAddress);
```

Comment:
I should make this a `connectedPlayers` list and take the `connected:0|1` property of the response AS differentiator. Between the 'disconnected' and 'forget' client notifications the player will still be reported here, but as disconnected.